cmd/user-agent/src/store/modules/user.js

```diff
@@ -4,11 +4,14 @@ Copyright SecureKey Technologies Inc. All Rights Reserved.
 SPDX-License-Identifier: Apache-2.0
 */
 
-import {WalletManager, Messenger} from "../../pages/chapi/wallet";
+import {Messenger, WalletManager} from "../../pages/chapi/wallet";
 import * as Aries from "@trustbloc/agent-js-worker";
 
 // TODO message type domain needs to be finalized
-const msgServices = [{name: 'request-peer-did', type: 'https://didcomm.org/peerdidrequest/1.0/message'}]
+const msgServices = [
+    {name: 'request-peer-did', type: 'https://didcomm.org/peerdidrequest/1.0/message'},
```

Comment:
does it make sense to use `trustbloc.github.io` instead of `didcomm.org`? This would make it clear that this message type is a TrustBloc thing, not an Aries thing.

Reply:
I am using type from 'hub-router',  otherwise message handler will not catch incoming messages. 
Yes, we have to revisit message types.